src/main/java/uk/gov/hmcts/probate/config/notifications/NotificationTemplates.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.probate.config.notifications;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+import org.springframework.validation.annotation.Validated;
+import uk.gov.hmcts.probate.model.ApplicationType;
+
+import javax.validation.Valid;
+import java.util.Map;
+
+@Getter
+@Setter
+@Component
+@Validated
```

Comment:
Does this class need to be @Validated?

Reply:
https://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-external-config.html#boot-features-external-config-validation